File: rb/test/exists_test.rb
# OktiasBakery SDK exists test

require "minitest/autorun"
require_relative "../OktiasBakery_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = OktiasBakerySDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- OktiasBakery SDK exists test

local sdk = require("oktias-bakery_sdk")

describe("OktiasBakerySDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
